package.json

```diff
@@ -94,5 +94,8 @@
     "start:stage": "BACKEND_DOMAIN=https://treeherder.allizom.org node ./node_modules/neutrino/bin/neutrino start --presets ./neutrino-custom/development.js",
     "test": "node ./node_modules/neutrino/bin/neutrino test --presets ./neutrino-custom/test.js",
     "test:watch": "node ./node_modules/neutrino/bin/neutrino test --watch --presets ./neutrino-custom/test.js"
+  },
+  "resolutions": {
+    "d3": "^5"
```

Comment:
I looked at Renovate and it turns out they support keeping `resolutions` up to date, so long as the version matches that in `dependencies`. As such, could you change this to `5.7.0`? The `d3` entry in `dependencies` will also need changing to `5.7.0`. Could you also move the resolution section to above `dependencies`? (Makes it easier to see it's overriding things when looking at `package.json`)

Reply:
Oh and #3367 can be closed by this PR, so could you add a `"Fixes #3367."` to the commit message, so GitHub automatically closes?

Reply:
Done.